### PR TITLE
2018 Toyota Sienna XLE support

### PR DESCRIPTION
**Pull request by daehahn**
Tested with Adam Schroeder, but still need to refine STATIC_MESSAGES(Headlight and Cruise Fault error exist). Cabana drive is here - 10/2 may be before get fingerprint and last one should be  fingerprinted but with stock DSU.

Cabana 10/02/2018 - New_DBC1542137910614.csv
https://community.comma.ai/cabana/?route=3b6e00dde85c3462%7C2018-10-02--06-02-02&max=5&url=https%3A%2F%2Fchffrprivate.blob.core.windows.net%2Fchffrprivate3%2Fv2%2F3b6e00dde85c3462%2F9ef1b01f6b24cf2337584526dee582d3_2018-10-02--06-02-02

Cabana 10/02/2018 - New_DBC1542139106955.csv
https://community.comma.ai/cabana/?route=3b6e00dde85c3462%7C2018-10-02--06-11-39&max=17&url=https%3A%2F%2Fchffrprivate.blob.core.windows.net%2Fchffrprivate3%2Fv2%2F3b6e00dde85c3462%2F02e0ea85d553c2386ed033622e7d05d1_2018-10-02--06-11-39

Cabana 10/30/2018 - New_DBC1542139190492.csv
https://community.comma.ai/cabana/?route=3b6e00dde85c3462%7C2018-10-30--01-51-37&max=35&url=https%3A%2F%2Fchffrprivate.blob.core.windows.net%2Fchffrprivate3%2Fv2%2F3b6e00dde85c3462%2Fdcb085c54c9c0c041a4591c6150089d4_2018-10-30--01-51-37

### CODE REVIEW

**Comment by rbiasini** [comment]
@daehahn , can you please make a PR for `opendbc` repo first? note that the generated dbc file should be autogenerated with `generator.py`. See how the other dbc files are made

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -22,39 +23,39 @@ class ECU:
...
+  (0x344, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.RAV4, CAR.COROLLA, CAR.HIGHLANDER, CAR.HIGHLANDERH, CAR.SIENNA), 0,   5, '\x00\x00\x01\x00\x00\x00\x00\x50'),
   (0x365, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.HIGHLANDERH), 0,  20, '\x00\x00\x00\x80\x03\x00\x08'),
   (0x365, ECU.DSU, (CAR.RAV4, CAR.COROLLA, CAR.HIGHLANDER), 0,  20, '\x00\x00\x00\x80\xfc\x00\x08'),
   (0x366, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.HIGHLANDERH), 0,  20, '\x00\x00\x4d\x82\x40\x02\x00'),
-  (0x366, ECU.DSU, (CAR.RAV4, CAR.COROLLA, CAR.HIGHLANDER), 0,  20, '\x00\x72\x07\xff\x09\xfe\x00'),
-  (0x470, ECU.DSU, (CAR.PRIUS, CAR.LEXUS_RXH), 1, 100, '\x00\x00\x02\x7a'),
-  (0x470, ECU.DSU, (CAR.HIGHLANDER, CAR.HIGHLANDERH, CAR.RAV4H), 1,  100, '\x00\x00\x01\x79'),
-  (0x4CB, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.RAV4, CAR.COROLLA, CAR.HIGHLANDERH, CAR.HIGHLANDER), 0, 100, '\x0c\x00\x00\x00\x00\x00\x00\x00'),
+  (0x366, ECU.DSU, (CAR.RAV4, CAR.COROLLA, CAR.HIGHLANDER, CAR.SIENNA), 0,  20, '\x00\x72\x07\xff\x09\xfe\x00'),
+  (0x470, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH), 1, 100, '\x00\x00\x02\x7a'),
```
Comment (rbiasini) [inline selfdrive/car/toyota/values.py]: why this change to RAV4H?

**Comment by vanillagorillaa** [comment]
Closing and @hyperactive32 will open with the most up to date commits
